### PR TITLE
Website copywriting edits

### PR DESCRIPTION
**Pull request by jazifresh**
Okay, Take 2, notes:
-Please verify that the integrity of what Buoy is trying accomplish is preserved in these edits.
-All the elements of the page I edited are still there but re-arranged (more technical aspects towards the end, larger simplify concept up from)
-I removed language that keyed in on the 911 alternative because I think it's distracts from explaining Buoy simply (I understand this aspect is a huge motivator for this project and will hopefully be used in this manner but it sounds like more than a cop watch or cop response alternative)
-I also recommend not using the current screenshot talking about a police encounter or at least one that includes very explicitly what the Alerter wants the response to be. Right now it's left open to interpretation (need backup for what exactly? do they want more filmers, are the calling for a direct confrontation, physical opposition?)
-The copy in the readme (can't remember where I saw it) is awesome and direct as well. I think it does a great job detailing Buoy as an alternative to police assistance. I don't know how this page and the readme function in the overall picture or where they'll be applied but I edited this page with someone like me in mind (specifically someone not a programmer)

### CODE REVIEW

File: about/index.md [hunk 1]
```diff
@@ -5,16 +5,16 @@ weight: 10
 
 <img src="http://i.imgur.com/cADD79P.png" alt="Buoy: Tell your friends where you are and what you need." align="right" width="300" />
 
-Buoy is a free and open source, decentralized tool designed to quickly connect users with trusted friends, family members, advocates, and other allies in times of need.
+Buoy is a crisis response platform that connects families, friends, and teams during emergency situations. 
 
-Buoy is a community-based crisis response system for any emergency in which someone needs help but might not want to -- or can't -- call 911.
+Buoy is a free tool designed to quickly to quickly alert and communicate with a prescribed group of people when assistance is needed.
```
Comment (fabacab) [inline about/index.md:10]: There are duplicated words here: "to quickly to quickly"

File: about/index.md [hunk 2]
```diff
@@ -5,16 +5,16 @@ weight: 10
...
 
-Buoy allows users to pre-define a custom network of trusted friends and allies. In an emergency, the user's Response Team can be alerted with one touch. Each crisis alert provides Responders with a real-time map of the Alerter's and each others' locations, a chat room in which they can all coordinate a response, and media (photo, video, audio) sharing capabilities.
+Buoy allows users to to create customized networks of trusted friends and allies and to plan responses for emergency situations. In the case of an emergency, the user's Response Team can be alerted with one touch. The alert creates a real-time map of each team member's location, a live chat to coordinate a response, and live photo, audio, and video streaming capabilities.
```
Comment (fabacab) [inline about/index.md:12]: Technically, there's only video streaming. Photo sharing exists but isn't streamed. This is a technical distinction that matters to some people (me) but most people won't really understand the nuance. Is there a way to more clearly describe this without being technically inaccurate?

**Comment by fabacab** [comment]
This is a great edit, thanks so much!

The only issues I found with it are [a typo (duplicated words) in one line](https://github.com/betterangels/buoy/pull/138#discussion-diff-67597326) and a [somewhat technically inaccurate description](https://github.com/betterangels/buoy/pull/138#discussion-diff-67597342) on another (see the line notes on commit b6cda6f1f3401b1a571a24bcb93397e8651ed7c8, above).

You also referred to some more general things:

> The copy in the readme (can't remember where I saw it) is awesome and direct as well. I think it does a great job detailing Buoy as an alternative to police assistance. I don't know how this page and the readme function in the overall picture or where they'll be applied but I edited this page with someone like me in mind (specifically someone not a programmer)

Let me try to clarify the differences between these pages.

The About page you edited in commit b6cda6f1f3401b1a571a24bcb93397e8651ed7c8 ultimately gets displayed on the [Buoy website's About page, here](http://buoy.maymay.net/about/#main). I'm not sure what readme you're talking about, but if you're talking about [this file](https://github.com/betterangels/buoy/blob/develop/README.markdown), then it gets displayed on the [Buoy GitHub project's homepage, here](https://github.com/betterangels/buoy/#readme).

The difference is mostly about audience, since the two files are used in different contexts:
- The purpose of the About page on the Buoy website is to describe Buoy as a project to laypeople and the general public.
- The purpose of the README file (all caps) is to describe Buoy as a project to people interested in contributing to the project.

In other words, the README file gets distributed with the project's source code itself (whenever someone downloads Buoy, they get a copy of the README file), so it needs to be useful for people who are comfortable working with software. The About page does not need to do this; it's only ever used as website copy.

There is a second ["readme" file (no caps), here](https://github.com/betterangels/buoy/blob/develop/readme.txt). This file's contents ultimately gets displayed on [the Buoy plugin page at the WordPress plugin directory](https://wordpress.org/plugins/buoy/). Its purpose is to describe Buoy as a plugin to WordPress site administrators who are using WordPress already and may want to add Buoy's features to their websites.

If this seems like a lot, that's because it is. Each of these contexts are different and have slightly different (though potentially overlapping) audiences.

> I also recommend not using the current screenshot talking about a police encounter or at least one that includes very explicitly what the Alerter wants the response to be.

Sounds reasonable. Can you either [open a new issue](https://github.com/betterangels/buoy/issues/new) describing a better alternative or include another commit in this pull request to address this?

Thanks and great work so far!

**Comment by hurtstotouchfire** [comment]
@meitar I can't find this file anywhere anymore. Is this obsolete?

**Comment by fabacab** [comment]
> I can't find this file anywhere anymore. Is this obsolete?

This is a PR against the Buoy project website, so you'll find [this in the `gh-pages` branch](https://github.com/betterangels/buoy/blob/gh-pages/about/index.md).

**Comment by hurtstotouchfire** [comment]
:-/ why does this repo have a gh-pages branch? shouldn't that be over on https://github.com/betterangels/betterangels.github.io which is actually a github pages website repo?

**Comment by fabacab** [comment]
> :-/ why does this repo have a gh-pages branch? shouldn't that be over on https://github.com/betterangels/betterangels.github.io which is actually a github pages website repo?

Because the GitHub Pages feature can be used for both an organization- and a repository-level website. The `gh-pages` *branch* in the Buoy *repository* is the website for Buoy (https://betterangels.github.io/buoy/). The code in the `betterangels.github.io` *repository* (in the `master` branch) is what creates the Better Angels website (https://betterangels.github.io/).

File: about/index.md [hunk 3]
```diff
@@ -5,16 +5,16 @@ weight: 10
...
+In order to make this useful tool as accessible as possible to admins and developers of various skill levels, Buoy is implemented as a WordPress plugin. The platform integrates with other free software projects and services such as mapping with OpenStreetMap, video conferencing through Jitsi Meet, and offers advanced real-time location tracking and push notifications via HTML5 APIs.
 
-We are dedicated to free and open source software principles, and believe in the importance of technically-implemented decentralization in order to equitably distribute power. We have an explicitly anti-capitalist, anti-oppression, prison abolitionist, pro-community agenda.
+We are dedicated to free and open source software principles and believe in the importance of technically-implemented decentralization in order to equitably distribute power. We have an explicitly anti-capitalist, anti-oppression, prison abolitionist, pro-community agenda and are commited to incorporting those values into everything we do.
```
Comment (hurtstotouchfire) [inline about/index.md:20]: typo here too: `incorporating`

File: about/index.md (hunk 1 above)
Comment (hurtstotouchfire) [reply about/index.md:10]: Addressed in https://github.com/betterangels/buoy/pull/188/commits/7271f8c6ceb7ac7572de3d4db5220faaa832b842

File: about/index.md (hunk 3 above)
Comment (hurtstotouchfire) [reply about/index.md:20]: Addressed in d174069.

File: about/index.md (hunk 2 above)
Comment (hurtstotouchfire) [reply about/index.md:12]: @meitar does 0209d1970c0e53e1ef3e95430d9688fede4aacf9 suit you?

**Pull request closed without merging**